src/content/1.1/code/agda/snippet01.agda

```diff
@@ -1 +1 @@
-f : A -> B
\ No newline at end of file
+f : A → B
```

Comment:
Do you prefer to avoid unicode?

Reply:
I do not have strong opinion on this.

Pros of ASCII:
1. some Agda experts like Conor McBride promote -no-unicode style
2. it might be easier for programmers (who are target audience) to see ASCII arrows -> rather than unicode ones →

Cons:
3. some Agda experts prefer Unicode e.g. Martin Escardo, Philip Wadler, etc.
4. standard library uses Unicode heavily 

I am slightly for ASCII due to 2. WDYT?

Reply:
We agreed to go for unicode.